Add release note for `physicalNetworkName` for OVN-Kubernetes secondary networks (#89471)
openshift/openshift-docs: release_notes/ocp-4-18-release-notes.adoc

You still must restart any workloads that use the network attachment definition for the network changes to take effect for those pods.

[id="ocp-release-notes-networking-ovn-kubernetes-secondary-network-localnet-physical-network-name_{context}"]

> ==== OVN-Kubernetes secondary network support for sharing a physical network between localnets

Reviewer: ```suggestion
==== OVN-Kubernetes secondary network support for sharing a physical network between localnet topologies 
```
Same suggestion as here:

> When configuring a secondary network that uses the `localnet` topology, you can now specify a physical network name with the `physicalNetworkName` field. By default, this value is set to the name of the localnet. Multiple localnets can share the same physical network.

Reviewer: ```suggestion
When configuring a secondary network that uses the `localnet` topology, you can now specify a physical network name with the `physicalNetworkName` field. By default, this value is set to the name of the localnet. Multiple localnet topologies can share the same physical network.
```
I did not see the word "localnet" used in plural like this, so I would suggest adding "topologies".

> For more information, see xref:../networking/multiple_networks/secondary_networks/creating-secondary-nwt-ovnk.a
> doc#configuration-ovnk-network-plugin-json-object_configuring-additional-network-ovnk[Configuring a secondary network that uses the localnet topology].

Reviewer: ```suggestion
For more information, see xref:../networking/multiple_networks/secondary_networks/creating-secondary-nwt-ovnk.adoc#configuration-ovnk-network-plugin-json-object_configuring-additional-network-ovnk[OVN-Kubernetes network plugin JSON configuration table].
```
* Deleting space between `.a doc` to fix the link

* Also, I was first confused by the link text, I see writers only use the name of the modules for link texts, but when looking at guidelines, I did not actually find anything that would prohibit this :smile: but for the sake of consistency, I would consider adding the link text as the name of the module. 

* In the linked table I also did not see the `physicalNetworkName` mentioned, is that maybe being added with another PR?

Author: @eromanova97, that content lives in: https://github.com/openshift/openshift-docs/pull/87994
